src/components/common/Actions.vue

```diff
@@ -122,7 +122,7 @@ export default class Actions extends Mixins(FilingTemplateMixin, LegalApiMixin)
   private async onClickSaveResume (): Promise<void> {
     this.setIsSavingResuming(true)
     const filing = await this.buildFiling()
-    await this.saveFiling(filing, true)
+    await this.saveFiling(filing)
```

Comment:
I think you should wrap this statement in a try-catch block, and unwrap the methods in legal-api-mixin. Then here you can display some error dialog. (Error dialogs are future so you could just alert() here for now.)
Basically, I think Actions.vue should display the error, not the mixin.

Reply:
Makes sense, much easier to handle in the component itself than the mixin, will do!